gravitee-apim-portal-webui-next/src/app/api/api-details/api-tab-subscriptions/subscriptions-details/subscriptions-details.component.html

```diff
@@ -46,6 +46,12 @@
         [apiKeyConfigUsername]="detailsVM.result.apiKeyConfigUsername" />
     </div>
   }
+
+  @if (detailsVM.result && detailsVM.result.consumerConfiguration !== undefined) {
```

Comment:
```suggestion
  @if (!!detailsVM.result?.consumerConfiguration) {
```
WDYT in case it is falsy?